test/test_autograd.py

```diff
@@ -8887,6 +8887,102 @@ def fn(x1, x2):
         torch.autograd.gradcheck(fn, [inp_r, inp_c], check_forward_ad=True)
         torch.autograd.gradcheck(fn, [inp_c, inp_r], check_forward_ad=True)
 
+class TestAutogradMultipleDispatch(TestCase):
+    def test_autograd_multiple_dispatch_registrations(self, device):
+        t = torch.randn(3, 3, device=device, requires_grad=True)
+        # using _test_autograd_multiple_dispatch.one
+        out = torch._test_autograd_multiple_dispatch(t)
+        grad = torch.randn(3, 3, device=device)
+        out.backward(grad)
+
+        if 'cuda' not in device:
+            # bogus default gradient registered for Autograd is grad + 1
+            self.assertEqual(t.grad, grad + 1)
+        else:
+            # bogus gradient registered for AutogradCUDA is grad * 2
+            self.assertEqual(t.grad, grad * 2)
+
+        # test registered AutogradNestedTensor formula
```

Comment:
This is a duplicate of `test_autograd_composite_implicit_and_dispatch_registration` no?

Reply:
Hm not quite, I registered a different formula for `_test_autograd_multiple_dispatch.one`, the composite implicit test checks `_test_autograd_multiple_dispatch.two`. I was just trying to check that multiple registrations work in this test while the next test checks a single registration + composite implicit, I can remove this part if it's unecessary though!

Reply:
Ho right. I got confused by the two overloads :D

Reply:
Probably worth one-liners above the two functions noting the distinction (basically just the content in your reply to Alban)